### PR TITLE
Bug hunting

### PR DESCRIPTION
**Pull request by Stearnzy**
Add GET method for Top Rated button

File name corrected from '.rb.rb' to '.rb'

New Cassettes
  - Movie search for ' '
  - Movie search for random text

Changed search_movie cassette to use pre-existing movie_search cassette, deleted search_movie cassette

Wrote sad path tests and flashes
  - User clicks search button without entering anything
  - User searches for space or random text, page displays 'No results found'
  - Test written for clicking login without entering information

### CODE REVIEW

File: app/controllers/movies_controller.rb [hunk 1]
```diff
@@ -1,6 +1,11 @@
 class MoviesController < ApplicationController
   def index
-    @titles = MovieApiService.movie_search(params[:movie_title])
+    if params[:movie_title] != ''
+      @titles = MovieApiService.movie_search(params[:movie_title])
+    else
+      flash[:error] = 'Search field cannot be blank'
```
Comment (stillsheryl) [inline app/controllers/movies_controller.rb:6]: Nice addition to improve the user experience!

File: app/controllers/sessions_controller.rb [hunk 2]
```diff
@@ -22,4 +20,15 @@ def destroy
...
     redirect_to root_path
   end
+
+  private
+
+  def error_message
+    if params[:password].blank? && params[:email].blank?
+      flash[:error] = 'Fields cannot be blank.'
+    else
+      flash[:error] = 'Invalid email or password.'
```
Comment (stillsheryl) [inline app/controllers/sessions_controller.rb:30]: Nice thought to keep it vague to prevent the hax0rs

**Review by stillsheryl — APPROVED** [review]
Great job with edge case testing!

**Review by Eternal-Flame085 — APPROVED** [review]
Looks good

**Pull request merged**